tests/test_orient_sync.py

```diff
@@ -95,7 +95,7 @@ def testCommandLine(self):
         with tempfile.TemporaryDirectory() as tmpdir:
             # Save the simulation object into STAR and MRCS files
             starfile_out = os.path.join(tmpdir, "save_test.star")
-            starfile_in = os.path.join(DATA_DIR, "sample_relion_data.star")
+            starfile_in = os.path.join(DATA_DIR, "sample_particles_relion31.star")
```

Comment:
Yes, I figured wherever we're getting particles from a star file (outside of testing the 3.0 version), we should use 3.1

Reply:
Okay, the comment at the top ot the 31.star just made me want to ask.  Thanks

Reply:
Yeah, actually the comment was there from when I thought the new file would be much more different, but it now represents the same information

On balance, I'll remove that comment to prevent confusion